tools/build_chain.sh

```diff
@@ -457,6 +457,8 @@ generate_config_ini()
     ;log level INFO DEBUG TRACE
     Level=${log_level}
     MaxLogFileSize=209715200
+    ; control log auto_flush
+    FLUSH=true
```

Comment:
Flush

Reply:
All configurations related to log has been modified to low case.